### DISCUSSION TITLE
Change Iam arn

### DISCUSSION THREAD

**Discussion by ghost** [amzn/selling-partner-api-models#3155] [General] [answered]
I need to change the IAM ARN in the app registration but the field is disabled. Is there an easy way to do this ? 

Since my app is, not published yet, I tried to delete the app registration and create another with the new Iam ARN but there's no way to remove the app registration. 

How can I fix this ? Thanks

**Comment by JordanForeman** [comment, marked as answer]
Had a similar use case. I had to contact support, and they did this for me. IIRC it requires an internal DBAdmin request on Amazon's end and they have no intention of making this field editable 😞

**Reply by ghost** [reply]
Thanks for the help.

At least, they could let remove the app.

I would maybe create another one and rename this one "deleted" but I might have a couple deleted.

**Comment by Kinggrass** [comment]
Would also need this feature to delete them over the Seller Central Frontend.

I am only giving some Feedback here maybe someone on Amazon side is reading this.

**Comment by JordanForeman** [comment]
**Update:** 

:wave:  heya @mlefebvre-deblecx! You might find this announcement of particular interest: amzn/selling-partner-api-docs#1600 

> As of October 20th, 2021, developers can change the IAM ARN associated with their application without contacting Amazon for help

**Reply by ghost** [reply]
@JordanForeman Thanks a lot, I'll be really helpful. Now it's only missing the delete button.

Is there a way to subscribe to those announcements ? I couldn't find how to.

**Reply by JordanForeman** [reply]
I believe that you can get notified of discussions in general; I get notifications for announcements as well as question. As I'm continuing to integrate with this API, I've found it useful to stay abreast of developments as well as the struggles others are having 😅 

![image](https://user-images.githubusercontent.com/1462087/139919660-c9da1a77-1b00-47a1-9879-d543be155040.png)
